stages/org.osbuild.coreos.platforms

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python3
+"""
+Process all necessary operations related to CoreOS
+platforms is crucial and specific to CoreOS. This step
+is essential for CoreOS exclusively.
+"""
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "platform": {
+      "description": "The artifactory platform name",
+      "type": "string"
+    },
+    "platforms_json": {
+        "description": "The path where platforms.json is stored",
+        "type": "string"
+    },
+    "bootpath": {
+      "type": "string",
+      "description": "The mounted location of the boot filesystem tree where the BLS entries and grubconfig  will be under ./loader/entries/*.conf",
+      "pattern": "^(mount|tree):\/\/\/",
+      "examples": ["tree:///boot", "mount:///", "mount:///boot"],
+      "default": "tree:///boot"
+    },
+    "deployment": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["osname", "ref"],
+      "properties": {
+        "osname": {
+          "description": "Name of the stateroot to be used in the deployment",
+          "type": "string"
+        },
+        "ref": {
+          "description": "OStree ref to create and use for deployment",
+          "type": "string"
+        },
+        "serial": {
+          "description": "The deployment serial (usually '0')",
+          "type": "number",
+          "default": 0
+        }
+      }
+    }
+  }
+},
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+},
+"mounts": {
+  "type": "array"
+}
+"""
+import glob
+import json
+import os
+import osbuild.api
+import shutil
+import sys
+
+from urllib.parse import urlparse
+from osbuild.util import ostree
+
+
+def copy_file(source_path, destination_path):
```

Comment:
This seems generic enough it belongs in `osbuild.utils`

Reply:
I agree in principal but if we move it to a generic helper we need to clean it up a bit and add tests

Reply:
I dropped it as a function, and kept is just the built in one. If no error code is handle, doesn't make sense to have a func for it